### ISSUE TITLE
Blender 2.93: broken compilation

### ISSUE THREAD

**Issue by tobiasBora** [NixOS/nixpkgs#129092]
**Describe the bug**
On unstable, blender has been updated to 2.93 (the new, shining, LTS version). However, it does not compile due to its dependency opencolorio-2.0.1, meaning that I can't install blender on unstable.

EDIT: This may be linked with https://github.com/NixOS/nixpkgs/issues/126726, and this recent PR https://github.com/NixOS/nixpkgs/pull/127522 made by @veprbl seems to be the one that breaks blender.

**To Reproduce**
1.  Update your nixos-unstable channel (I'm using the commit 59ef8087a3d)
2.  Install blender

See error:
```
building '/nix/store/w57bv43wn3a11v6bphsn2qbm5fian74s-opencolorio-2.0.1.drv'...
unpacking sources
unpacking source archive /nix/store/xns2qwzrd2ilsqrmd7818f22lgysbyx8-source
source root is source
patching sources
configuring
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF [...]
-- The CXX compiler identification is GNU 10.3.0
-- The C compiler identification is GNU 10.3.0
-- Detecting CXX compiler ABI info
[...]
-- Found GLUT: /nix/store/91b165nnnz5f3wnxg6dzly4fqahl4bw9-freeglut-3.2.1/lib/libglut.so  
-- GLVND supported
-- Performing Test HAVE_SSE2
-- Performing Test HAVE_SSE2 - Success
-- Setting SOVERSION to '2.0' as none was specified.
CMake Error at /nix/store/hmnywaai8pbahlxriayriwfdviv8nzg4-expat-2.4.1-dev/lib/cmake/expat-2.4.1/expat.cmake:74 (message):
  The imported target "expat::expat" references the file

     "/nix/store/hmnywaai8pbahlxriayriwfdviv8nzg4-expat-2.4.1-dev/lib/libexpat.so.1.8.1"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "/nix/store/hmnywaai8pbahlxriayriwfdviv8nzg4-expat-2.4.1-dev/lib/cmake/expat-2.4.1/expat.cmake"

  but not all the files it references.

Call Stack (most recent call first):
  /nix/store/hmnywaai8pbahlxriayriwfdviv8nzg4-expat-2.4.1-dev/lib/cmake/expat-2.4.1/expat-config.cmake:35 (include)
  share/cmake/modules/Findexpat.cmake:33 (find_package)
  share/cmake/modules/FindExtPackages.cmake:29 (find_package)
  CMakeLists.txt:239 (include)


-- Configuring incomplete, errors occurred!
See also "/build/source/build/CMakeFiles/CMakeOutput.log".
error: builder for '/nix/store/w57bv43wn3a11v6bphsn2qbm5fian74s-opencolorio-2.0.1.drv' failed with exit code 1
error: 1 dependencies of derivation '/nix/store/w8kh64l3w2knzghq2kmfa8sr9rwzwwfs-blender-2.93.0.drv' failed to build
error: 1 dependencies of derivation '/nix/store/18mmq5cx58brq2svs1pd7kp3yqdwaffq-system-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/0yf2jkq5jgva96sss28smzvk357xws2h-nixos-system-bestos-21.11pre-git.drv' failed to build
```

**Notify maintainers**
@jonascarpay @goibhniu @veprbl @rytone

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.37, NixOS, 21.11pre-git (Porcupine)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.4pre20210601_5985b8b`
 - channels(root): `"nixos-unstable-21.11pre299739.59ef8087a3d, nixos-21.03pre267299.5ff6700bb82"`
 - channels(me): `"nixos-unstable-21.11pre299739.59ef8087a3d, local-nixos-unstable-21.05pre283883.d235056d6d6, home-manager, local-nixpkgs-unstable-21.03pre267409.f5daa8ab31f, nixos-20-03-20.03.3324.929768261a3, test"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by InternetUnexplorer** [comment]
I believe this will be fixed in #128997.

**Comment by tobiasBora** [comment]
Cool, good to see that people are working on it. I can confirm that blender 2.93 seems to work fine on this commit, thanks for the tip.

**Comment by veprbl** [comment]
Temporary fixed by #128808
